net/cryfs/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                1.0
+PortGroup                 github 1.0
+PortGroup                 cmake 1.1
+PortGroup                 active_variants 1.1
+
+github.setup              cryfs cryfs 0.10.2
+github.tarball_from       releases
+
+license                   LGPL-3
+
+categories                net fuse
+platforms                 darwin
+
+maintainers               @jolan78 openmaintainer
+
+description               Cryptographic filesystem for the cloud
+long_description          CryFS encrypts your files, so you can safely store them anywhere.\
+                          It works well together with cloud services like Dropbox, iCloud,\
+                          OneDrive and others
+
+homepage                  https://www.cryfs.org/
+
+checksums                 rmd160  9abdb5e59263cf8ea978470f4c6a4086ddd8a860\
+                          sha256  004c7fdf537062e197aaa0bcdafd2682904a279802441bb6063cb72c1fba52d2\
+                          size    10187912
+
+extract.mkdir             yes
+
+depends_build             port:openssl\
+                          port:python36\
+                          port:boost\
+                          path:/usr/local/include/osxfuse/fuse.h:osxfuse
+
+depends_lib               port:curl\
+                          port:libomp\
+                          path:/usr/local/lib/libosxfuse.2.dylib:osxfuse
+
+cmake.build_type          Release
+universal_variant         no
+
+configure.args-append    -DBoost_USE_MULTITHREADED=on
+
+variant no_version_check description {Doesn't check online for updates and security vulnerabilities} {
+    configure.args-append -DCRYFS_UPDATE_CHECKS=off
+}
+
+variant static_boost description {Link boost statically (requires -no_static variant of boost)} {
+    require_active_variants boost {} no_static
+}
+
+if {![variant_isset static_boost]} {
+    depends_lib-append port:boost
+    configure.args-append -DBoost_USE_STATIC_LIBS=OFF
+}
+
+
+variant system_osxfuse description {Prefer osxfuse installed from osxfuse.github.io} {
+    configure.cxxflags-append  -I/usr/local/include
```

Comment:
Macports does not rely on `/usr/local/ `. See https://trac.macports.org/wiki/FAQ#usrlocal

Reply:
Ok, I removed the system_osxfuse variant.